src/extensions/instant_server.rs

```diff
@@ -40,6 +41,24 @@ impl Default for InstantServer {
 
 impl InstantServer {
     pub fn new() -> Self {
+        InstantServer::config(
+            r#"
+            [policy_module]
+            operator_num = 1
+            [[policy_module.create_entry]]
+            key_name = "_policy_module:init:accept_all_tasks"
+            value = "true"
+            
+            [remote_storage]
+            operator_num = 1
+            
+            [registry]
+            operator_num = 1
+            "#,
+        )
+    }
+
+    pub fn config(user_init_config: &str) -> Self {
```

Comment:
so it is not configuring an existing one, but creating a new one with the config? Maybe we can reflect this in the func name?